### PR TITLE
AO3-4566 Update Gems for Ruby upgrades

### PR DESCRIPTION
**Pull request by zz9pzza**
https://otwarchive.atlassian.net/browse/AO3-4566

Looking at this I would just check that things work as the things changing look as if they are used every where eg mysql, memcached etc

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -11,7 +10,7 @@ gem 'rails', '3.2.22.2'
 
 # Database
 # gem 'sqlite3-ruby', :require => 'sqlite3'
-gem 'mysql2', '0.3.10'
+gem 'mysql2', '0.3.20'
```
Comment (shalott) [inline Gemfile]: didn't we already change mysql2 to something higher? I might be totally misremembering...

**Comment by shalott** [comment]
Gone grey

File: Gemfile (hunk 1 above)
Comment (zz9pzza) [inline Gemfile]: We have a pull request for it but it has yet to be merged.

https://github.com/otwcode/otwarchive/pull/2292

**Comment by sarken** [comment]
Grey again

**Comment by shalott** [comment]
Gone grey again

**Pull request merged**